index.html

```diff
@@ -277,7 +300,8 @@ <h3>Contact me</h3>
         <fieldset id="fs-frm-inputs">
           <input type="text" name="name" id="full-name" placeholder="Full name" maxlength="30" required>
           <input type="email" name="_replyto" id="email-address" placeholder="Email address" required>
-          <textarea rows="5" name="message" id="message" placeholder="Enter text here" maxlength="500" required></textarea>
+          <textarea rows="5" name="message" id="message" placeholder="Enter text here" maxlength="500"
+            required></textarea>
```

Comment:
As you see in the screenshot below the from still not centered so please fix that
![wrong](https://user-images.githubusercontent.com/58376020/126868236-0b45db09-c19a-46fe-b94a-e95b8c6d0406.png)